Dockerfile

```diff
@@ -24,7 +24,7 @@ WORKDIR /opt/wps
 # (2) Install RavenWPS in editable mode
 # Have to uninstall the ravenpy installed by conda so the re-install with
 # binaries work.  Same problem as in the Makefile.
```

Comment:
Sweet !!!  By the way, Makefile also need to undo the `pip install ravenpy --install-option="--with-binaries"`

Reply:
All good. Once It looks like it's running on dockerhub, I'll push a hotfix.

Reply:
Actually, now that I think of it. This is still the non-conda way of installing/running `ravenpy` so it's not technically obsolete. Are we abandoning pure python approaches completely then?

Reply:
I guess we'll still have to keep the pip way but this specific pip way is broken so that's why you guys had to do it the conda way.  So the current pip hack in Makefile is broken ... for the Makefile.

Reply:
It's not broken insofar as it's a bit esoteric/brittle. I'll push a fix for this in the coming week, before we tag a new version. Good eye.